show-grid/README.metadata.json

```
@@ -0,0 +1,46 @@
+{
+    "category": "MapViews, SceneViews and UI",
+    "description": "Display coordinate system grids including Latitude/Longitude, MGRS, UTM and USNG on a map view. Also, toggle label visibility and change the color of grid lines and grid labels.",
+    "formal_name": "ShowGrid",
+    "ignore": false,
+    "images": [
+        "show-grid.png"
+    ],
+    "keywords": [
+        "coordinates",
+        "degrees",
+        "graticule",
+        "grid",
+        "latitude",
+        "longitude",
+        "mgrs",
+        "minutes",
+        "seconds",
+        "usng",
+        "utm",
+        "Grid",
+        "LatitudeLongitudeGrid",
+        "MGRSGrid",
+        "MapView",
+        "SimpleLineSymbol",
+        "TextSymbol",
+        "UsngGrid",
+        "UtmGrid"
+    ],
+    "language": "kotlin",
+    "redirect_from": "",
+    "relevant_apis": [
+        "Grid",
+        "LatitudeLongitudeGrid",
+        "MGRSGrid",
```

Should this one be in Pascal case like the others?
```suggestion
        "MgrsGrid",
```